### ISSUE TITLE
"Submodules to execute" drop-down in "Language selection" is missing from the tab order

### ISSUE THREAD

**Issue by jsmaniac** [racket/drracket#71]
In the "Language selection" window (Ctrl+L), the drop-down menu "Submodules to execute" cannot be activated via the tab key or the arrow keys. The focus jumps directly from "Impose constant definitions" to "Other languages" (with tab) or "Constructor" (with the down arrow).

![Screenshot of the "Language selection" window](https://cloud.githubusercontent.com/assets/299872/18320171/0764f540-7529-11e6-9fa0-b52d68904eba.png)
